fetch.js

```diff
@@ -12,7 +12,7 @@ if (typeof process !== 'undefined' && process) {
     module.exports = fetch
   } else {
     // Node.js or Electron browser process
-    module.exports = require('node-fetch')
+    module.exports = typeof fetch === 'undefined' ? require('node-fetch') : fetch.bind(globalThis)
```

Comment:
nit: should we just require fetch.native from here?

Reply:
@mvayngrib no, we should just remove all this an unify in a follow-up